markdown/bitburner.charityaugmentationareas.md

```diff
@@ -0,0 +1,49 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [bitburner](./bitburner.md) &gt; [CharityAugmentationAreas](./bitburner.charityaugmentationareas.md)
+
+## CharityAugmentationAreas type
+
+Charity Augmentation Areas.
+
+**Signature:**
+
+```typescript
+type CharityAugmentationAreas =
```

Comment:
Type should not be pluralized.

e.g. 
```ts
const augArea: CharityAugmentationAreas = "hacking";
```
In usage it is clear that something with this type is referring to a single area, not multiple areas, so the plural type name is confusing.